packages/insomnia-components/components/list-group/unit-test-result-item.js

```diff
@@ -0,0 +1,54 @@
+// @flow
+import * as React from 'react';
+import styled from 'styled-components';
+import SvgIcon from '../svg-icon';
+import ListGroupItem from './list-group-item';
+import UnitTestResultTimeStamp from './unit-test-result-timestamp';
+import UnitTestResultBadge from './unit-test-result-badge';
+
+type Props = {|
+  item: Object,
+|};
+
+const StyledResultListItem: React.ComponentType<{}> = styled(ListGroupItem)`
+  > div:first-of-type {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+
+    > *:not(:first-child) {
+      margin: var(--padding-xs) 0 var(--padding-xs) var(--padding-sm);
+    }
+  }
+
+  code {
+    background-color: var(--hl-xs);
+    padding: var(--padding-sm) var(--padding-md) var(--padding-sm) var(--padding-md);
+    color: var(--color-danger);
+    display: block;
+    margin: var(--padding-sm) 0 0 0;
+  }
+
+  p {
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    flex-grow: 1;
+  }
+`;
+
+const UnitTestResultItem = ({ item }: Props) => {
+  return (
+    <StyledResultListItem>
+      <div>
+        <UnitTestResultBadge result={item.err.message} />
```

Comment:
Then updating this usage:

```suggestion
        <UnitTestResultBadge failed={!!item.err.message} />
```